src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -5989,9 +5987,8 @@ private static void CombineExtensionMethodArguments(BoundExpression receiver, An
                 if (!hasError)
                 {
                     var isFixedStatementExpression = SyntaxFacts.IsFixedStatementExpression(node);
-                    var isInsideNameof = this.EnclosingNameofArgument != null;
                     Symbol accessedLocalOrParameterOpt;
-                    if (IsNonMoveableVariable(receiver, out accessedLocalOrParameterOpt) == isFixedStatementExpression && !isInsideNameof)
+                    if (IsNonMoveableVariable(receiver, out accessedLocalOrParameterOpt) == isFixedStatementExpression && !IsInsideNameof)
```

Comment:
>!IsInsideNameof [](start = 122, length = 15)

It looks like this condition is redundant because it is checked by an enclosing ```if```.